libs/screen/targetoverrides.ts

```diff
@@ -48,6 +57,48 @@ namespace brick {
         return ((screen.height - textOffset) / lineHeight()) >> 0
     }
 
+    /**
+     * Number of columns
+     */
+    //%
+    export function columnCount(): number {
+        return ((screen.width - textOffset) / font.charWidth) >> 0
+    }
+
+    /**
+     * Show text on the screen on a specific line and starting at a column and the selected print style.
+     * @param text the text to print on the screen, eg: "Hello world"
+     * @param line the line number to print the text at (starting at 1), eg: 1
+     * @param column the column number to print the text at (starting at 1), eg: 1
+     * @param printStyle print style black on white or white on black
+     */
+    //% blockId=screenPrintString block="show string %text|at line %line||and column %column|with style %printStyle"
```

Comment:
```suggestion
    //% blockId=screenPrintString block="show string $text|at line $line||column $column|style $printStyle"
```

nit: for new blocks we avoid using % as it causes issues with translations when e.g. we need to swap the order of words for things to make sense. We only leave it on existing blocks to avoid breaking those existing translations unnecessarily

we also typically avoid conjunctions when possible, e.g. `col [] row []`, `x [] y []`, etc. The `with` in `with style` can stay if you want probably, though I'd lean towards eliding it